apps/crn-frontend/src/events/options.ts

```diff
@@ -17,13 +17,15 @@ export type GetEventListOptions = GetListOptions &
       }
   ) & {
     userId?: string;
+    groupId?: string;
   };
 
 export const getEventListOptions = (
   currentTime: Date,
   past: boolean,
   { searchQuery = '', currentPage = 0, pageSize = CARD_VIEW_PAGE_SIZE } = {},
   userId?: string,
+  groupId?: string,
```

Comment:
I am not sure about this interface as we add more options it will become less usable. Should this be an object? Potentially extending the search query one?